src/camvidlog2/bioclip/cli.py

```
@@ -0,0 +1,150 @@
+    db: Annotated[Path, typer.Option()] = Path("tmp.feather"),
+    num: Annotated[int, typer.Option("--num", "-n", min=0)] = 10,
+    roll: Annotated[int, typer.Option("--rolling", "-r", min=0)] = 0,
+    json: Annotated[Path | None, typer.Option()] = None,
+):
+    video_embeddings = data_load(db)
+    if video_embeddings is None:
+        raise ValueError("Unable to load database")
+
+    if json:
+        if queries:
+            raise ValueError("Cannot use --json and [QUERIES]")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)